x/pki/keeper/msg_server_add_noc_x_509_ica_cert.go

```diff
@@ -101,11 +103,14 @@ func (k msgServer) AddNocX509IcaCert(goCtx context.Context, msg *types.MsgAddNoc
 		msg.CertSchemaVersion,
 	)
 
-	// Add a NOC certificate to the list of NOC certificates with the same VID
-	k.AddNocIcaCertificate(ctx, certificate)
+	// Add to the global list of certificates
+	k.AddAllCertificate(ctx, certificate)
```

Comment:
Should we also add to AllCertificates by Subject and/or SKID?

Reply:
Added one more index of all certificates by subject